gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/tasks/CreateManifestTask.java

```diff
@@ -402,7 +471,7 @@ private void validateProjectVersion() {
                     task.getProductType().set(ext.getProductType());
                     task.setManifestFile(new File(project.getBuildDir(), "/deployment/manifest.yml"));
                     task.getProductDependencies().set(ext.getProductDependencies());
-                    task.getProductDependenciesConfig().set(project.provider(ext::getProductDependenciesConfig));
+                    task.setConfiguration(project.provider(ext::getProductDependenciesConfig));
```

Comment:
maybe for a follow-up - store a property<configuration> in the extension instead